docs/release-notes/cloud/cloud-2021-09.md

```diff
@@ -0,0 +1,84 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud September 2021.
+included_jira_versions: ['2021.Q3.5', '2021.Q3.6', '2021.Q4.1']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/3.7.0
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/3.9.12
+---
+
+# Cloud September 2021
+
+These release notes are for the Codacy Cloud updates during September 2021.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   [<span class="skip-vale">codacy-clang-tidy</span>](https://github.com/codacy/codacy-clang-tidy) is now also published as a macOS binary. (CY-5053)
+-   Added the following plugins to [<span class="skip-vale">codacy-eslint</span>](https://github.com/codacy/codacy-eslint):
+    -   [<span class="skip-vale">eslint-plugin-i18n-json</span>](https://www.npmjs.com/package/eslint-plugin-i18n-json) (CY-5029)
+    -   [<span class="skip-vale">eslint-plugin-suitescript</span>](https://www.npmjs.com/package/eslint-plugin-suitescript) (CY-4918)
+    -   [<span class="skip-vale">eslint-config-xo-typescript</span>](https://www.npmjs.com/package/eslint-config-xo-typescript) (CY-4890)
+    -   [<span class="skip-vale">@vue/cli-plugin-babel</span>](https://www.npmjs.com/package/@vue/cli-plugin-babel) (CY-4864)
+    -   [<span class="skip-vale">eslint-plugin-sort-keys-fix</span>](https://www.npmjs.com/package/eslint-plugin-sort-keys-fix) (CY-4763)
+    -   [<span class="skip-vale">eslint-plugin-playwright</span>](https://www.npmjs.com/package/eslint-plugin-playwright) (CY-4751)
+
+## Bug fixes
+
+-   Fixed an issue that caused a redirect to different repository settings when enabling the GitHub integration for a repository. (CY-5056)
+-   Fixed an issue that caused the PMD option `ignoreUsings` to be ignored when calculating duplication for C#. (CY-5026)
+-   Fixed Codacy Coverage Reporter not finding some LCOV coverage reports automatically. (CY-5015)
+-   Fixed an issue that caused the error `Failed to analyze file` when CodeNarc identified issues using code patterns that had no associated message. (CY-4974)
+-   Improved the code pattern names for Cppcheck. (CY-4887)
```

Comment:
Could we add 
```suggestion
-   Improved the code pattern names for Cppcheck. (CY-4887)
-   Improved the severity for issues detected by Stylelint
``` ?
https://codacy.atlassian.net/browse/CY-4851

Reply:
Yes, thanks for mentioning this @inesgomas! This issue wasn't picked up automatically because it is of type "Task". I'm adding it manually now. :+1: